src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: auto;
+  padding: 10px;
+  width: calendar-width(
+    $days-in-a-row, 
+    $square-side, 
+    $distance-between-squares
+  );
+  display: flex;
+  flex-wrap: wrap;
+  gap: $distance-between-squares;
+
+  &__day {
+    width: $square-side;
+    height: $square-side;
+    background-color: #eee;
+    border: 1px solid black;
+    text-align: center;
+    line-height: $square-side;
+    transition-duration: $execution-time;
```

Comment:
You should specify transition properties directly
Default is all, but it could affect old browsers' performance 
Yurii wrote about it previously